headers/types/files/wan.h

```diff
@@ -1,14 +1,99 @@
 #ifndef HEADERS_TYPES_FILES_WAN_H_
 #define HEADERS_TYPES_FILES_WAN_H_
 
-// For now, there is only the top-level header for WAN files.
+#include "../common/generic.h"
 
-// Directly loaded from memory as they are stored. Names comes from the pmd_wan project.
-#pragma pack(push, 1)
+enum wan_sprite_type {
+    PropsUI = 0, // for object and UI objects,
```

Comment:
Enum variant names are global to the whole program. Can we make all these more specific? They should also be `SCREAMING_SNAKE_CASE` per convention. Perhaps `WAN_SPRITE_PROPS_UI`, etc.?